frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -353,6 +364,21 @@ export const VMSchedulingList: React.FC<VMSchedulingListProps> = ({
   const tolerationsWrapperCount = (vmiLikeWrapper?.getTolerations() || []).length;
   const affinityWrapperCount = getRowsDataFromAffinity(vmiLikeWrapper?.getAffinity())?.length;
 
+  const watchResource = React.useMemo(() => {
+    return {
+      kind: kubevirtReferenceForModel(KubeDeschedulerModel),
+      isList: true,
+    };
+  }, []);
+  const [resourceList] = useK8sWatchResource<any>(watchResource);
+  const isDeschedulerInstalled = resourceList.length > 0;
+  const isVMliveMigratable =
+    isVMRunningOrExpectedRunning(vm, vmi) &&
+    vmi?.status?.conditions?.filter((obj) => obj.type === 'LiveMigratable').length > 0;
```

Comment:
this condition will fetch you the 'LiveMigratable' condition, and should check for the obj.status === 'true'
btw we have a function to fetch us the vmi conditions by type, it's located right here: 
https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/selectors/vmi/basic.ts#L22